pkg/apis/upgrade/v1alpha1/register.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the Upgrade v1alpha1 API group
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=kubeedge/pkg/apis/upgrade
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=upgrade.kubeedge.io
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	SchemeBuilder      runtime.SchemeBuilder
+	localSchemeBuilder = &SchemeBuilder
+	AddToScheme        = localSchemeBuilder.AddToScheme
+)
+
+const (
+	// GroupName is the group name use in this package.
+	GroupName = "upgrade.kubeedge.io"
```

Comment:
```suggestion
	GroupName = "operations.kubeedge.io"
```
`operations` may be a better group name, we can include other operation api in future.
cc @zhu733756 @gy95 @wackxu  ptal

Reply:
+1

Reply:
+1

Reply:
done

```
group: "upgrade.kubeedge.io" -> operations.kubeedge.io"
kind: Upgrade -> NodeUpgrade
```